src/main/java/org/eclipse/emfcloud/jackson/databind/ser/EMapSerializer.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright (c) 2019-2021 CS GROUP and others.
```

Comment:
```suggestion
 * Copyright (c) 2022 CS GROUP and others.
```

Reply:
Copy paste error

Should look better now...